README.md

```diff
@@ -13,6 +13,7 @@ This repo can be installed on either the same VM as the main LORIS codebase, or
 On <u>Ubuntu</u>, DICOM toolkit will be installed by the imaging install script (step 4 below). This script will _apt-get install dcmtk_.   
 
 For <u>CentOS</u>: Dependency installation notes are included in the [LORIS wiki](https://github.com/aces/Loris/wiki/Imaging-Database) Imaging Setup page, Section 1 (installing codebase)</b>.
+As of the release date, this includes a transcript for [CentOS installation](https://github.com/aces/Loris/wiki/CentOS-Imaging-installation-transcript) and notes on dependencies including [DICOM toolkit](https://github.com/aces/Loris/wiki/CentOS-Imaging-installation-transcript#7-install-dicom-toolkit).
 
```

Comment:
It is greta to add this here.
My only comment is as I go to the Wiki, the DICOM section, I would not keep the two lines on _export_ statement or would make it clearer that below are two possible examples